inc/Engine/HealthCheck/CacheDirSizeCheck.php

```diff
@@ -146,14 +142,13 @@ private function get_dir_size( $dir ) {
 	 * Sends a notification to our endpoint with the type of directory
 	 *
 	 * @since 3.3.5
-	 * @author Remy Perona
 	 *
 	 * @param string $dir_type Type of directory.
 	 * @return void
 	 */
 	private function send_notification( $dir_type ) {
 		wp_safe_remote_post(
-			WP_ROCKET_WEB_MAIN . '/api/wp-rocket/cache-dir-check.php',
+			rocket_get_constant( 'WP_ROCKET_WEB_MAIN' ) . 'api/wp-rocket/cache-dir-check.php',
```

Comment:
Same thing here. Let's move this to a property and set it in the constructor.

Reply:
https://github.com/wp-media/wp-rocket/pull/2616/commits/9e95cf8cd5eb809236b17004a12c3b43715b4c56